src/vaadin-combo-box-mixin.html

```diff
@@ -552,7 +552,7 @@
 
     /** @private */
     _closeOrCommit() {
-      if (this.autoOpenDisabled && !this.opened) {
+      if (this.autoOpenDisabled && !this.opened && !this.loading) {
```

Comment:
Nit: not related to this PR, but `loading` could be a bit dangerous to rely on as it persists after leaving the field with pending requests **and is not removed afterwards.**
![loading](https://user-images.githubusercontent.com/29372419/96970106-db270580-151b-11eb-8e41-b27adc521d09.gif)
**Definitely not a blocker**

Reply:
In this example, has the data provider returned the data? Data provider should set loading to false on return.

```javascript
            if (Object.keys(this._pendingRequests).length === 0) {
              this.loading = false;
            }
```

Reply:
https://github.com/vaadin/vaadin-combo-box/blob/5007fcc182ed59fb1813e79e24349612561a781b/src/vaadin-combo-box-data-provider-mixin.html#L168-L170

Reply:
This is happening on the `Lazy loading` in the `Basic` if you quickly type the value and leave the field. It doesn't pass the first [condition](https://github.com/vaadin/vaadin-combo-box/blob/5007fcc182ed59fb1813e79e24349612561a781b/src/vaadin-combo-box-data-provider-mixin.html#L152) as there is no pending requests anymore. 
But as said this is happening on master as well, so not caused by this PR.

Reply:
Created issue #963